proto/celestia/qgb/v1/types.proto

```diff
@@ -15,7 +15,7 @@ message BridgeValidator {
   string evm_address = 2;
 }
 
-// Valset is the EVM Bridge Multsig Set, each qgb validator also
+// Valset is the EVM Bridge Multsig Set, each Blobstream validator also
```

Comment:
[Suggestion-optional] For the sake of consistency in comments and Godocs, it would be good to have consistent capitalization i.e., using either "blobstream" or "Blobstream". The "Blobstream" resembles a struct/type name. Same for "qgb" vs "QGB".

Reply:
"Blobstream" when referring to the product, and "`blobstream`" when referring to the binary, is what I'm basing documentation off of

Reply:
yes, and also sometimes in the logs and panics, it's better to use small capitalization

Reply:
Thank you for the clarification. As long as the usage of "blobstream" or "Blobstream" adheres to this semantic, I'm on board with it.